homeassistant/components/conversation/__init__.py

```diff
@@ -47,22 +47,6 @@
 )
 
 
-CONFIG_SCHEMA = vol.Schema(
```

Comment:
Also remove `DATA_CONFIG` constant and any code that touches `hass.data[DATA_CONFIG]`